future/name-resolution/002-nr-host-synchronization/README.md

```diff
@@ -128,68 +140,78 @@ All this output is a little overwhelming. You can click on each command to see t
+Okay, let's see if we can make sense of all this by following the timestamps.
+
+What's a timestamp? Well, the first thing in each line of the `tcpdump` output is the **exact** time that the packet was seen. These timestamps look a little something like this: `19:59:53.972283`. This means that when the machine running `tcpdump` saw a packet, it looked at its watch and found that the time was approximately 8:00pm local time (this is in 24-hour time). But computers have *very* precise clocks, and the *exact* time of this packet was 7:59pm and 53 seconds and 972 hundred-thousanths of a second! This kind of precision might seems a little ridiculous, but when packets are flying around the network quickly, having very exact time for each one is essential to help us understand which ones came first and which ones came next.
+
+Since we have precise timestamps being reported for each packets by `host-c` and `host-f` (and note that the clocks on both of these machines are synchronized), we can look back and forth between the output from both machines and put the packets in chronological order. So let's look at the timestamps across all the output: what's the first thing you see happening?
 
 > **host-c:**
 > 19:59:53.972283 IP (tos 0x0, ttl 255, id 50541, offset 0, flags [DF], proto UDP (17), length 58)
     6.0.0.103.5353 > 224.0.0.251.5353: [bad udp cksum 0xe799 -> 0x628b!] 0 A (QM)? host-f.local. (30)
 
-Before `host-c` can send out the ICMP packets, it needs to know the IP address of the machine its sending them to. We know that `host-c` is configured through `/etc/nsswitch.conf` to use multicast to lookup any `.local` address. Any address within the `224.0.0.0/4` subnet is designated as a multicast address; specifically, `224.0.0.251` is reserved for multicast DNS requests. We can also see that it's using the port `5353`, which is also reserved for multicast DNS requests.
+So remember that we started this process with a `ping` command on `host-c`. Now, before `host-c` can send out a ping packet, it needs to know the IP address of the machine its sending them to. All the computer knows from the command we typed in was that we wanted to send a ping to `host-f.local`. So `host-c`'s first job is to figure out what IP address is associated with `host-f.local`.
+
+Next, we know that `host-c` is configured through `/etc/nsswitch.conf` to use "multicast" to look up any `.local` address.
+
+> But, what is this "multicast" anyway? We haven't really talked about it. Strictly speaking, a multicast IP address is any IP address within the `224.0.0.0/4` subnet. Any time a message is sent to an IP address in this range, it will be "multicasted". For an Ethernet network like the one that our hosts are both on, this really just means that the packet will be broadcast to every machine on that network.
```

Comment:
I love this explanation. Would it be helpful to link to some documentation? 

https://www.iana.org/assignments/multicast-addresses/multicast-addresses.xhtml#:~:text=Host%20Extensions%20for%20IP%20Multicasting,255.255.